### ISSUE TITLE
Some questions about features extracting.

### ISSUE THREAD

**Issue by ascetic-monk** [fjchange/MIST_VAD#3]
Thanks for your awesome work and complete code. I think it is one of the most complete framework in WSVAD. 
The code is of high quality, but after reading your code, I am a little confused about two issues:
1. What is the setting of feature extracting before generate PLs? Is it the same as the test phase? Do you feed the whole video as input or feed every single clip into the backbone and concat them?
2. The official link of Shanghai Tech dataset lapsed. Can you give another solid link or upload your h5-file？

**Comment by fjchange** [comment]
1. For UCF-Crime, we extract features from pretrained C3D (trained on Sport1M released by FAIR) and  I3D (trained on Kinetics-400, tranlated to pytorch verison by in https://github.com/hassony2/kinetics_i3d_pytorch
2. You may need to contact with the author, but I also provide the pre-extracted features in [OneDrive](https://1drv.ms/f/s!Ai48CHyipiNUkFTHTQGze7QLY1Fn)

**Comment by fjchange** [comment]
I uploaded the SH_Frame.h5 and corresponding annotations on BaiduYun.

link：https://pan.baidu.com/s/1sQUGXj-BnLDGczWuGkBWdA 
code：kym5

**Comment by chengengliu** [comment]
> I uploaded the SH_Frame.h5 and corresponding annotations on BaiduYun.
> 
> link：[pan.baidu.com/s/1sQUGXj-BnLDGczWuGkBWdA](https://pan.baidu.com/s/1sQUGXj-BnLDGczWuGkBWdA)
> code：kym5

Hi Jichang, how did you make SH_Frames.h5 file?

**Comment by yashika-git** [comment]
@fjchange, I just want to do an evaluation of the code on the test set of UCF-Crime. Could you please tell if the code expects the extracted I3D features as input or shall the input be raw videos from which it will itself extract the I3D features?
Thank you!
